### PR TITLE
Update releasing notes in README.rst

### PR DESCRIPTION
**Pull request by ukanga**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -100,3 +100,32 @@ To check out the documentation for pyxform do the following::
...
+
+Releasing pyxform
+=================
+
+On a release branch from latest upstream master, (assumes you are working on your fork of pyxform):
+
+1. Update ``CHANGES.txt`` with issues closed from the previous tagged release e.g. https://github.com/XLSForm/pyxform/compare/v0.14.1...master.
+2. Update ``README.rst``, ``setup.py``, ``pyxform/__init__.py`` with the new release version number.
+3. Commit, push branch, and initiate a pull request. Wait for tests to pass.
+4. Prepare a draft release, copy the changes noted in ``CHANGES.txt`` to the draft release. Set version number and the tiltle for the release should include the date of release.
```
Comment (yanokwa) [inline README.rst]: s/tiltle/title

File: README.rst [hunk 2]
```diff
@@ -100,3 +100,32 @@ To check out the documentation for pyxform do the following::
...
+5. When all tests are passing on pull request, squash merge the pull-request.
```
Comment (yanokwa) [inline README.rst]: s/pull-request/pull request/

File: README.rst [hunk 3]
```diff
@@ -100,3 +100,32 @@ To check out the documentation for pyxform do the following::
...
+6. Checkout to master branch an pull in latest upstream master.::
```
Comment (yanokwa) [inline README.rst]: s/to/the
s/an/and 
s/master./master

File: README.rst [hunk 4]
```diff
@@ -100,3 +100,32 @@ To check out the documentation for pyxform do the following::
 Change Log
 ==========
 `Changelog <CHANGES.txt>`_
+
+Releasing pyxform
+=================
+
+On a release branch from latest upstream master, (assumes you are working on your fork of pyxform):
```
Comment (yanokwa) [inline README.rst]: I'd make this the first step. I don't think you need the parathentical. 

1. Checkout a release branch from latest upstream master.

**Comment by lognaturel** [comment]
Looks great to me. Is the idea that you want to try these before approving and merging, @yanokwa?

**Pull request merged**